config/crd/bases/anywhere.eks.amazonaws.com_cloudstackmachineconfigs.yaml

```diff
@@ -38,6 +38,48 @@ spec:
             description: CloudStackMachineConfigSpec defines the desired state of
               CloudStackMachineConfig
             properties:
+              ISOAttachment:
+                description: ISOAttachment attach an ISO rom which has been previously
```

Comment:
Reads more clearly without "ISOAttachment" in the description string.   Also, "attaches".